python/lsst/obs/base/makeRawVisitInfo.py

```diff
@@ -48,12 +48,13 @@
 
 
 class MakeRawVisitInfo(object):
-    """Base class functor to make a VisitInfo from the FITS header of a raw image
+    """Base class functor to make a VisitInfo from the FITS header of a raw image.
 
-    A subclass will be wanted for each camera. Subclasses should override
-    - setArgDict: the override can call the base implementation,
-                    which simply sets exposure time and date of observation
-    - setDateAvg
+    A subclass will be wanted for each camera. Subclasses should override:
+
+    - ``setArgDict``, The override can call the base implementation,
```

Comment:
Because I was confused. In this case, single backticks works (sphinx correctly finds the function). Double backticks are for monospace font, without trying to link to a python object.

Reply:
Thank you. That is very useful information. I need a good reference manual for this stuff.

Reply:
@jonathansick 's page on the dev guide is quite good: https://developer.lsst.io/docs/py_docs.html